networkit/cpp/centrality/KadabraBetweenness.h

```diff
@@ -0,0 +1,237 @@
+/*
+ * KadabraBetweenness.h
+ *
+ * Created on: 18.07.2018
+ * 		 Author: Eugenio Angriman, Alexander van der Grinten
+ */
+
+#ifndef KADABRA_H_
+#define KADABRA_H_
+
+#include <atomic>
+
+#include "../auxiliary/PQVector.h"
+#include "../base/Algorithm.h"
+#include "../graph/Graph.h"
+
+namespace NetworKit {
+
+class Status {
```

Comment:
In contrast to the original code, we only use `Status` on thread zero. Can we get rid of this? Or would that require too much effort?

Reply:
Yes we can get rid of it but I doubt that it would bring significant performance improvements, so I would prefer to leave it as a future development.

Reply:
OK, let's do this after the initial merge.